tpm2/tpm2.go

```diff
@@ -1469,6 +1469,30 @@ func (cmd FlushContext) Execute(t transport.TPM, s ...Session) (*FlushContextRes
 // FlushContextResponse is the response from TPM2_FlushContext.
 type FlushContextResponse struct{}
 
+// EvictControl is the input to TPM2_EvictControl.
+// See definition in Part 3, Commands, section 28.5
+type EvictControl struct {
+	// TPM_RH_OWNER or TPM_RH_PLATFORM+{PP}
+	AuthHandle       handle `gotpm:"handle,auth"`
+	ObjectHandle     handle `gotpm:"handle"`
+	PersistentHandle TPMIDHObject
```

Comment:
nit: can you make the type `TPMIDHPersistent`? This might require adding this after the definition for TPMIDHObject in structures.go

```go
// TPMIDHPersistent represents a TPMI_DH_PERSISTENT.
// See definition in Part 2: Structures, section 9.5.
type TPMIDHPersistent = TPMHandle
```

The reason to make it a different alias of `TPMHandle` is so that if we ever promote these types from aliases to actual types with their own validation based on the TPM spec, everything is already using the right names.